### ISSUE TITLE
The playbook fails with "Unable to retrieve file contents" on Ansible 2.4 

### ISSUE THREAD

**Issue by nmasse-itix** [nmasse-itix/threescale-cicd#45]
The playbook fails with this error:
```raw
TASK [nmasse-itix.threescale-cicd : include_tasks] *********************************************************************************************************************************************************
fatal: [TENANT-admin.3scale.net]: FAILED! => {"reason": "Unable to retrieve file contents\nCould not find or access '/opt/ansible/threescale-cicd/support/docker/steps/find_goswagger.yml'"}
	to retry, use: --limit @/opt/ansible/threescale-cicd/support/docker/deploy-api.retry
```

When this error appears, it seems this is tied to an old version of Ansible that has a bug in it:
```raw
# ansible --version
ansible 2.4.2.0
  config file = /opt/ansible/threescale-cicd/support/docker/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Oct 30 2018, 23:45:53) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

**Comment by nmasse-itix** [comment]
It seems the Ansible version packaged by default on CentOS and RHEL is 2.4.2 and it has this bug. 
Our Travis CI tests are running against two versions of Ansible 2.4.6 and 2.7.5. So the bug must have been resolved between 2.4.2 and 2.4.6.

**Comment by nmasse-itix** [comment]
I added the version check at the beginning of the playbook.
